app/controllers/candidate_interface/gcse/science/grade_controller.rb

```diff
@@ -42,20 +42,20 @@ def update
         end
       else
         track_validation_error(@gcse_grade_form)
-        render view_path
+        render view_path(update_path: true)
       end
     end
 
   private
 
-    def view_path
+    def view_path(update_path: false)
       if gcse_qualification? && current_qualification.award_year.nil?
         'candidate_interface/gcse/science/grade/awards_new'
       elsif gcse_qualification?
         'candidate_interface/gcse/science/grade/awards_edit'
       elsif current_qualification.award_year.nil?
         'candidate_interface/gcse/science/grade/new'
-      else
+      elsif update_path
```

Comment:
Shouldn't there also be an alternative path in case none of the conditions are satisfied.

Reply:
I did have one but rubocop kept flagging it as a 'duplicate branch body' but will revisit the whole statement